R/data_meredith.R

```diff
@@ -0,0 +1,353 @@
+#'   \item{rep_estimate}{}
+#'   \item{rep_hi}{}
+#'   \item{rep_lo}{}
+#'   \item{timestamp}{}
+#'  
+#' }
+#' @source See \url{https://github.com/fivethirtyeight/data/blob/master/congress-generic-ballot/README.md}
+"generic_topline"
+
+
+
+#' An Inconvenient Sequel
+#'
+#' The raw data behind the story
+#' "Al Gore’s New Movie Exposes The Big Flaw In Online Movie Ratings"
+#' \url{https://fivethirtyeight.com/features/al-gores-new-movie-exposes-the-big-flaw-in-online-movie-ratings/}.
+#'
+#' @format A data frame with 80053 rows representing movie ratings and 27 variables:
+#' \describe{
+#'   \item{timestamp}{The date at which the rating was recorded.}
+#'   \item{respondents}{The number of respondents in a category associated with a given timestamp.}
+#'   \item{category}{The subgroups of respondents differentiated by demographics like gender, age, and nationality.}
+#'   \item{link}{The website associated with a given category's responses.}
+#'   \item{average}{The average rating reported by a given category.}
+#'   \item{mean}{The mean rating reported by a given category.}
+#'   \item{median}{The median rating reported by a given category.}
+#'   \item{1_votes}{The count of votes denoting a rating of one that respondents gave.}
+#'   \item{2_votes}{The count of votes denoting a rating of two that respondents gave.}
+#'   \item{3_votes}{The count of votes denoting a rating of three that respondents gave.}
+#'   \item{4_votes}{The count of votes denoting a rating of four that respondents gave.}
+#'   \item{5_votes}{The count of votes denoting a rating of five that respondents gave.}
+#'   \item{6_votes}{The count of votes denoting a rating of six that respondents gave.}
+#'   \item{7_votes}{The count of votes denoting a rating of seven that respondents gave.}
+#'   \item{8_votes}{The count of votes denoting a rating of eight that respondents gave.}
+#'   \item{9_votes}{The count of votes denoting a rating of nine that respondents gave.}
+#'   \item{10_votes}{The count of votes denoting a rating of ten that respondents gave.}
+#'   \item{1_pct}{The percentage of votes denoting a rating of one that respondents gave.}
+#'   \item{2_pct}{The percentage of votes denoting a rating of two that respondents gave.}
+#'   \item{3_pct}{The percentage of votes denoting a rating of three that respondents gave.}
+#'   \item{4_pct}{The percentage of votes denoting a rating of four that respondents gave.}
+#'   \item{5_pct}{The percentage of votes denoting a rating of five that respondents gave.}
+#'   \item{6_pct}{The percentage of votes denoting a rating of six that respondents gave.}
+#'   \item{7_pct}{The percentage of votes denoting a rating of seven that respondents gave.}
+#'   \item{8_pct}{The percentage of votes denoting a rating of eight that respondents gave.}
+#'   \item{9_pct}{The percentage of votes denoting a rating of nine that respondents gave.}
+#'   \item{10_pct}{The percentage of votes denoting a rating of ten that respondents gave.}
```

Comment:
you'll need to update these variable names based on my comments in `process_data_meredith.R`